stock_valuation_layer_accounting_date/tests/test_stock_valuation_layer_accounting_date.py

```diff
@@ -17,12 +17,12 @@ class TestStockValuationStandard(TestStockValuationStandard):
     def setUp(self):
         super(TestStockValuationStandard, self).setUp()
 
-    @freeze_time("2022-12-02 23:00:00", tz_offset=9)
+    @freeze_time("2022-12-02")
     def test_svl_accounting_date_real_time(self):
         self.product1.product_tmpl_id.categ_id.property_valuation = "real_time"
```

Comment:
We shouldn't be changing the intention of the test.
```suggestion
    @freeze_time("2022-12-02 23:00:00")
    def test_svl_accounting_date_real_time(self):
        self.product1.categ_id.property_valuation = "real_time"
        self.env.user.tz = "Asia/Tokyo"
```